### PR TITLE
How-to/Get-Connected page

### LINKED ISSUE #401: Implement new /how-to/get-connected page
Wireframes here: https://www.figma.com/file/TfWk1iDHdlt7bSA3DjQuQv/Mobility-Marketplace?node-id=4606%3A20365&t=Xw6FPzkZQzHdUq6g-0

Copy here: https://docs.google.com/document/d/1uLjCTC0LEjaDtOivWQp0OFM0ZVIjULnr_cZ6-QgAi0k/edit#heading=h.hu6tpby747nd

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes #401 

## What this PR does

- This PR adds a new page to the site: https://deploy-preview-406--cal-itp-mobility-marketplace.netlify.app/how-to/get-connected, which has the same layout as https://deploy-preview-406--cal-itp-mobility-marketplace.netlify.app/how-to/go-contactless

- The Get Connected page has two tables. The large table has a fixed column on mobile: 
image

- This PR adds a link to the top nav to get to the `Get connected` page directly from the top. It moves the `Provider map` link from the top nav to the footer. Note: The Figma designs call for a redesign of the nav to include a dropdown menu (Products, leading to Get connected and Go contactless). This work will be done in the _next_ ticket, so as not to slow down deployment.
image
image

## Testing
- Tested on Macbook (Chrome, Firefox, Safari) and iPhone (Safari)

## Styles
- Adds utility classes: `text-normal`, `underline` for `h4`
- Table utility classes: `table-blue`, `border-blue`, `table-sticky-column` (add to `<td>` to make a sticky column), `table-caption-bottom`
- A custom width utility class with a breakpoint: `md-w-75` - meaning give 75% width at the `md` breakpoint and above. Used as an narrower alternative to `img-fluid` (which is 100% width all the time).

## To Dos

- [x] Responsive column
- [x] Remove border
- [x] Fix bullet pts
- [x] Add images
- [x] Add buttons
- [x] White link
- [x] Mobile: Table goes over the nav
- [x] Mobile: Table has a underline

### CODE REVIEW

**Comment by machikoyasuda** [comment]
@angela-tran @thekaveman This PR is ready for code review.

@segacy1 This PR is ready for design review.

Note: I talked with @thekaveman and @mrose914 earlier about the menu nav (specifically, adding a Products dropdown with 2 links in it). It's going to take a deep refactor / redesign of the code to make this possible, so we decided to add the Get connected page link to the top nav as a plain direct link for now, and implement the menu next in a separate pull request, tracked here #409

**Review by segacy1 — APPROVED** [review]

File: src/_data/how_to/get_connected.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        url: "what-is-connectivity"
+        paragraphs:
+          - text: From accepting mobile wallets for fare payments to giving the live location of a bus, transit is using more technology to improve the rider experience. Cellular data is the power behind this new technology. Types of devices may include&#58;
+          - text: <ul><li>Router</li><li>CAD/AVL</li><li>Camera feeds</li><li>Passenger WiFi</li></ul>
+          - text: As transit continues to modernize, more devices can be expected to leverage cellular data and connectivity.
+      - title: How do data plans work?
+        url: "how-do-data-plans-work"
+        paragraphs:
+          - text: Your technology can be connected through multiple SIMs (one per device) or a single SIM (one in a router).
+          - text: <u>Multiple SIMs</u>
```
Comment (thekaveman) [inline src/_data/how_to/get_connected.yml:14]: I feel like these would be more appropriate as `<h4>` styled as the underlined text.

See the [Usage Notes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u#usage_notes) on the `<u>` element, it doesn't seem right here:

> You should _not_ use `<u>` to underline text for presentation purposes, or to denote titles of books.

**Review by thekaveman — CHANGES_REQUESTED** [review]

File: src/_data/how_to/get_connected.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
...d">$30.00</td> <td class="border-dark-blue table-bordered">N/A</td> <td class="border-dark-blue table-bordered">N/A</td> </tr> <tr> <th class="table-dark-blue border-dark-blue table-sticky-column text-white table-bordered" scope="row">5 GB</th> <td class="border-dark-blue border-left-0 border-right-0">N/A</td> <td class="border-dark-blue border-left-0 border-right-0">N/A</td> <td class="border-dark-blue table-bordered border-left-0 border-right-0">N/A</td> <td class="border-dark-blue table-bordered">$28.00</td> <td class="border-dark-blue table-bordered">$21.21</td> <td class="border-dark-blue table-bordered">$25.00</td> </tr> <tr> <th class="table-dark-blue border-dark-blue table-sticky-column text-white table-bordered" scope="row">Unlimited</th> <td class="border-dark-blue table-bordered border-left-0 border-right-0">$34.00</td> <td class="border-dark-blue table-bordered border-left-0 border-right-0">$34.00</td> <td class="border-dark-blue table-bordered border-left-0 border-right-0">$29.99</td> <td class="border-dark-blue table-bordered">$50.00</td> <td class="border-dark-blue table-bordered">N/A</td> <td class="border-dark-blue table-bordered">$34.99</td> </tr> </tbody></table>
```
Comment (thekaveman) [inline src/_data/how_to/get_connected.yml:43]: This is rough, seems like sort of running against the usefulness of keeping everything in this data file. Is there a way to maintain this massive HTML table in HTML intead?

Comment (thekaveman) [reply src/_data/how_to/get_connected.yml:43]: Note this is nonblocking for this PR, more of a thought / something to look into later.

File: src/_data/how_to/get_connected.yml (hunk 1 above)
Comment (machikoyasuda) [reply src/_data/how_to/get_connected.yml]: Resolved: https://github.com/cal-itp/mobility-marketplace/pull/406/commits/4a3dbcae53d9387eb8340d3f3bdb80a36555f1ba

**Review by thekaveman — APPROVED** [review]

**Pull request merged**